client/src/index.jsx

```diff
@@ -54,28 +55,29 @@ class ApateezGallery extends React.Component {
     var appContext = this;
     // `${BASE_URL}/api/restaurants/${id}/gallery`
     axios.get(`http://localhost:3002/api/restaurants/${id}/gallery`)
```

Comment:
If you set the BASE_URL in the webpack config file, I think you don't have to hard code the url in the component files themselves.